### ISSUE TITLE
make cells selectable

### ISSUE THREAD

**Issue by pourquoi42** [pitamar/jupyter-module-loader#1]
typically, in a notebook there is all kinds of code that shouldn't be imported. It would be good to use tags on cells to mark those cells that actually need to be imported. I can create a PR if you let me contribute.

**Comment by pitamar** [comment]
Sure, sounds insteresting :slightly_smiling_face: 
Could you give me an example of the way you envision the use of tags for this?

**Comment by pourquoi42** [comment]
* individual cells can be marked with a tag
* in the nb metadata, there is a special key/value
=> only those cells will be run in the namespace. If the key is missing in the nb metadata, behaviour is as it is now

![image](https://user-images.githubusercontent.com/20537603/80710623-5371b880-8aef-11ea-81ec-20535854d675.png)

**Comment by pitamar** [comment]
Ah cool, I didn't know about that.
If that's the case then we could add a parameter to the `register()` function to specify what tags to import.
Something like this:
```
import jupyter_module_loader
jupyter_module_loader.register(tags=['tag1', 'tag2'])
```

If none are given it could revert to the default behavior of importing all the cells.
What do you think?

**Comment by pourquoi42** [comment]
yes, giving it as a parameter with .register() would be even better than configuring it in the notebook metadata. 
The ultimate solution would be also to detect dependent cells, but that's "slightly" more involved.

**Comment by pitamar** [comment]
Cool, do you want to try implementing this?

**Comment by pourquoi42** [comment]
sure, but I can only make a PR if you add me as collaborator, I guess.

**Comment by pitamar** [comment]
Really? that's odd, it's a public repository..
I tried from another Github account and it allows me to fork and create a new pull request:
![image](https://user-images.githubusercontent.com/4238626/80718687-23341500-8b03-11ea-8bcf-721809e03fa2.png)
